app/models/concerns/startdate_beyond_osra_establishment_validation.rb

```diff
@@ -0,0 +1,12 @@
+module StartdateBeyondOsraEstablishmentValidation
+	include ValidatorHelpers
+
+	OSRA_ESTABLISHMENT_DATE = Date.new(2012,04,01)
```

Comment:
Is there a file that Rails use to keep global variables? I think it is more tidy